wit/wasmcloud/lattice-control/lattice-control.wit

```diff
@@ -0,0 +1,560 @@
+      credentials: option<list<tuple<string, registry-credential>>>
+    }
+
+    record registry-credential {
+      /// The ID of the lattice on which this request will be performed
+      token: option<string>,
+
+      /// Username to use for HTTP basic auth
+      username: option<string>,
+
+      /// Password to use for HTTP basic auth
+      password: option<string>,
+
+      /// Type of registry (i.e. either "oci" or "bindle")
+      registry-type: option<string>,
+    }
+
+    /// Seek out a list of suitable hosts for a capability provider given a set of host label constraints.
+    ///
+    /// Hosts on which this provider is already running will not be among the successful "bidders" in the auction.
+    auction-provider: func(req: provider-auction-request) -> list<provider-auction-ack>;
+
+    /// Seek out a list of suitable hosts for an actor given a set of host label constraints
+    auction-actor: func(req: actor-auction-request) -> list<actor-auction-ack>;
+
+    /// Queries the list of hosts currently visible to the lattice.
+    ///
+    /// This is a "gather" oepration and so can be influenced by short timeouts, network partition events, etc.
+    get-hosts: func(lattice-id: string) -> list<host>;
+
+    /// Queries the contents of a host given the supplied 56-character unique ID
+    get-host-inventory: func(req: get-host-inventory-request) -> host-inventory;
+
+    /// Queries the lattice for the list of known/cached claims by taking the response
+    /// from the first host that answers the query.
+    get-claims: func(lattice-id: string) -> get-claims-response;
+
+    /// Publish a link definition into the lattice, allowing it to be cached and
+    /// delivered to the appropriate capability provider instances
+    advertise-link: func(req: advertise-link-request) -> ctl-operation-ack;
+
+    /// Requests the removal of a link definition.
+    ///
+    /// The definition will be removed from teh cache and the relevant capability providers
+    /// will be given a chance to de-provision any used resources
+    remove-link: func(req: remove-link-definition-request) -> ctl-operation-ack;
+
+    ///////////////////////
+    /// Actor Operations //
+    ///////////////////////
+
+    /// Instructs a given host to start the indicated actor
+    start-actor: func(cmd: start-actor-command) -> ctl-operation-ack;
+
+    /// Instructs a given host to scale the indicated actor
+    scale-actor: func(cmd: scale-actor-command) -> ctl-operation-ack;
+
+    /// Instructs a given host to perform a live update on the indicated actor
+    update-actor: func(cmd: update-actor-command) -> ctl-operation-ack;
+
+    /// Instructs a given host to stop the indicated actor
+    stop-actor: func(cmd: stop-actor-command) -> ctl-operation-ack;
+
+    ////////////////////////////////
+    // Link definition Operations //
+    ////////////////////////////////
+
+    /// Queries all c urrent link definitions in the specified lattice.
```

Comment:
Spelling: extra space between `c` and `urrent`

Reply:
- [ ] fix spelling